action/governance/finalizeProposal.go

```diff
@@ -298,7 +298,11 @@ func executeConfigUpdate(ctx *action.Context, proposal *governance.Proposal) err
 	}
 	err = ctx.GovernanceStore.WithHeight(ctx.Header.Height).SetStakingOptions(updatedGov.StakingOptions)
 	if err != nil {
-		return errors.Wrap(err, "Setup Fee Options")
+		return errors.Wrap(err, "Setup Staking Options")
+	}
+	err = ctx.GovernanceStore.WithHeight(ctx.Header.Height).SetEvidenceOptions(updatedGov.EvidenceOptions)
```

Comment:
Do we need this as part of this or, since this touches governance

Reply:
It is just fix for error message